entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/metamodel/ViewMetamodelImpl.java

```diff
@@ -88,6 +89,12 @@ public ViewMetamodelImpl(EntityMetamodel entityMetamodel, MetamodelBuildingConte
             ManagedViewTypeImplementor<?> managedView = context.getManagedViewType(viewMapping, null);
 
             managedViews.put(viewMapping.getEntityViewClass(), managedView);
+
+            Class<?> entityClass = viewMapping.getEntityClass();
+            if (!entitySources.containsKey(entityClass)) {
```

Comment:
@beikov I assume that the `*TypeImpl` (e.g. `ManagedViewTypeImplementor` above) class is meant to hold metadata about the entity/view that is gathered when entityViewManager is created. However I'm struggling to work out if the required metadata is already available at this stage via `managedView` or whether I'd need to implement the gathering of metadata about the entity, and create instances of `AbstractAttribute<?, ?>`